src/Pagination.jsx

```diff
@@ -272,11 +280,27 @@ export default class Pagination extends React.Component {
     let jumpNext = null;
     let firstPager = null;
     let lastPager = null;
+    let gotoButton = null;
 
+    const goButton = props.goButton || (props.showQuickJumper && props.showQuickJumper.goButton);
```

Comment:
> Why support `goButton` and `showQuickJumper.goButton` both?

I think `goButton` is follow `showQuickJumper` feature, so I add both. But maybe it break your specifications. I will remove it.

And I will try to support customized node. Good idea. :)